src/_data/groups.yml

```diff
@@ -1,4 +1,18 @@
 ---
+## Template set of params for easy copy/paste
+# - name:
+#   source:
+#   meetup_id:
+#   url:
+#   twitter_handle:
+#   slack:
+#   discord:
+#   youtube:
+#   twitch:
+#   github:
+#   website:
+#   email:
```

Comment:
Just a heads up https://github.com/MikeRogers0/Ruby-Meetup-Calendar/blob/main/lib/groups_sorter.rb - sometimes this file is auto formatted & comments might be not persist. 

I think maybe a `src/_data/groups.yml.sample` file showing the options could be the solution to this maybe?